api/types/types.go

```diff
@@ -202,22 +202,29 @@ type Info struct {
 	// LiveRestoreEnabled determines whether containers should be kept
 	// running when the daemon is shutdown or upon daemon start if
 	// running containers are detected
-	LiveRestoreEnabled bool
-	Isolation          container.Isolation
-	InitBinary         string
-	ContainerdCommit   Commit
-	RuncCommit         Commit
-	InitCommit         Commit
-	SecurityOptions    []string
-	ProductLicense     string `json:",omitempty"`
-	Warnings           []string
+	LiveRestoreEnabled  bool
+	Isolation           container.Isolation
+	InitBinary          string
+	ContainerdCommit    Commit
+	RuncCommit          Commit
+	InitCommit          Commit
+	SecurityOptions     []string
+	ProductLicense      string `json:",omitempty"`
+	DefaultAddressPools []NetworkAddressPool
+	Warnings            []string
 }
 
 // KeyValue holds a key/value pair
 type KeyValue struct {
 	Key, Value string
 }
 
+// NetworkAddressPool is a temp struct used by Info struct
```

Comment:
What do you mean by temp

Reply:
I mean this struct only used by Info struct , I see some other comments nearby like `PluginsInfo is a temp struct holding Plugins name` or `ExecStartCheck is a temp struct used by execStart` using the same word.